test/unit/main/DaprClient.test.ts

```diff
@@ -40,4 +40,9 @@ describe('DaprClient', () => {
       expect(values[0]).toEqual(values[1]);
     });
   });
+
+  it('should return dapr api token', () => {
```

Comment:
Please create a new category named "Dapr API Token" with 2 test:
1. This one to make sure we can communicate and see the options
2. Another one showing failure

P.S. I approved test runs but I think this will make the other tests fail since they require an API token which is now missing

Reply:
How should it fail ?

Reply:
By creating a client without the API token that should provide a forbidden error

Reply:
But that will require creating a new client in E2E testing and invoking a request which will return Forbidden.
I feel like i am testing dapr itself not the client, the main purpose of this PR is just adding the token to the header so I guess a more suitable test for the use case is checking if the token is being sent in the headers or not.

Reply:
Agreed, will approve! Thanks a lot for your collaboration 😊

Reply:
I guess we needed to remove `DAPR_API_TOKEN=securedtoken` from package.json